src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,18 @@
+package duke.task;
+public class Todo extends Task{
+    public String getTaskIcon() {
+        return "T";
+    }
+
+    public Todo(String description) {
+        super(description);
+    }
+    public String getDescription() {
+        int firstDivider = description.indexOf("todo");
+        String descTask = description.substring(firstDivider);
+//        int secondDivider = description.indexOf("/at");
+//        String descTask = description.substring(firstDivider + 5,secondDivider - 1);
+//        String duration = description.substring(secondDivider + 4);
```

Comment:
Avoid keeping commented-out code in the program. It reduces readability. If it is not required, consider removing it.